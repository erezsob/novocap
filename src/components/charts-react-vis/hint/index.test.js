import React from 'react';
import renderer from 'react-test-renderer';
import Hint from './index';

describe('<Hint />', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(<Hint title="TestTitle" value={{ y: 200 }} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
