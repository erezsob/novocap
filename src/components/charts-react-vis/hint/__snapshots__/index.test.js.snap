// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Hint /> renders correctly 1`] = `
<div
  className="hint"
>
  <small
    className="small"
  >
    TestTitle
  </small>
  <div
    className="value"
  >
    € 200.00
  </div>
</div>
`;
